packages/cli/src/cli.ts

```diff
@@ -1673,7 +1673,7 @@ function faucetCall(_: CLINetworkAdapter, args: string[]): Promise<string> {
     .then((faucetTx: any) => {
       return JSONStringify({
         txid: faucetTx.txId!,
-        transaction: generateExplorerTxPageUrl(faucetTx.txId!, new StacksTestnet()),
+        transaction: generateExplorerTxPageUrl(faucetTx.txId!.substr(2), new StacksTestnet()),
```

Comment:
It would be safer to check for the prefix before trimming, e.g. `faucetTx.txId.replace(/^0x/, '')`

Reply:
done!
```Some people, when confronted with a problem, think "I know, I'll use regular expressions." Now they have two problems.```